omega/omega-connector/omega-connector-grpc/src/main/java/org/apache/servicecomb/saga/omega/connector/grpc/LoadBalancedClusterMessageSender.java

```diff
@@ -124,14 +125,15 @@ public void close() {
   }
 
   @Override
-  public void send(TxEvent event) {
+  public AlphaResponse send(TxEvent event) {
+    AlphaResponse response = null;
```

Comment:
have to think about when this response will be returned, and the consequence of returning this response under such a scenario. apparently it's only possible when this thread is interrupted and throwing an exception is a better idea in such case.